app/views/handlers/primary_person_question.py

```diff
@@ -14,7 +14,16 @@ def _get_routing_path(self):
         return self.router.routing_path(section_id=self._current_location.section_id)
 
     def is_location_valid(self):
-        return self.router.can_access_location(self.parent_location, self._routing_path)
+        is_list_item_id_valid = (
```

Comment:
This is already done as part of `can_access_location`, but for list collectors, we do this with the `parent_location` but this is checking the child location since primary person's ID is in the URL before they are added vs normal list collector ID are generated on the POST.